api/android/api/src/main/jni/Android.mk

```diff
@@ -82,4 +93,9 @@ GSTREAMER_INCLUDE_CA_CERTIFICATES := no
 
 include $(GSTREAMER_NDK_BUILD_PATH)/gstreamer-1.0.mk
 
+#------------------------------------------------------
+# NDK cpu-features
+#------------------------------------------------------
+ifeq ($(ENABLE_TF_LITE), true)
```

Comment:
Could we eliminate this ifeq (99) and above ifeq (69)? For example, 

```
LOCAL_EXTRA_CFLAGS :=
ifeq ($(ENABLE_TF_LITE), true)
include $(LOCAL_PATH)/Android-tensorflow-lite.mk
LOCAL_EXTRA_CFLAGS += -DENABLE_TF_LITE=1
endif
...omission...
LOCAL_CFLAGS += LOCAL_EXTRA_CFLAGS 
```
How about that?

Reply:
added new flags as you commented (NNS_API_FLAGS and NNS_API_STATIC_LIBS)